### PR TITLE
chore: project addition by [ankit-0369] #1728

### PR DESCRIPTION
**Pull request by ankit-0369**
## Related Issue

this pr closes issue no #1728 .

## Description
this is a portfolio website which cann give idea about design using only html and css.

### CODE REVIEW

**Comment by ankit-0369** [comment]
I did not understand the reason of failure. can you suggest what can I do change @priyankarpal

File: DB/projects.json [hunk 1]
```diff
@@ -5407,5 +5407,22 @@
...
+    "LinkedIn": "https://www.linkedin.com/in/ankit0369/",
+    "Instagram": "https://www.instagram.com/_k.ankit__/",
+    "Twitter": "https://twitter.com/k_ankit1001"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/ankit-0369/portfolio-template",
+      "title": "portfolio-template",
+      "description": "this is a protfolio website which can be used to get an idea about animations and design of a portfolio website.",
+      "tech": ["html5", "css"]
```
Comment (priyankarpal) [inline DB/projects.json:5424]: change to `html`
```suggestion
      "tech": ["html", "css"]
```

File: DB/projects.json [hunk 2]
```diff
@@ -5407,5 +5407,22 @@
...
+    },
```
Comment (priyankarpal) [inline DB/projects.json:5425]: remove the comma `,`
```suggestion
    }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by ankit-0369** [comment]
done with changes @priyankarpal  new to github so thanks for ur suggestions

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**